test/roo/test_excelx.rb

```diff
@@ -133,6 +133,13 @@ def test_expand_merged_range
     end
   end
 
+  def test_expand_merged_range_doesnt_insert_nil_values
+    options = { expand_merged_ranges: true }
+    xlsx = roo_class.new(File.join(TESTDIR, "merged_ranges.xlsx"), options)
+
+    refute_includes xlsx.sheet_for(0).cells.values, nil, "`nil` was copied into the cells hash from an empty merged range"
+  end
```

Comment:
Can you also write a test case which verifies that issue posted in https://github.com/roo-rb/roo/issues/506 is actually solved

Reply:
Sure thing.

I originally added the same problem to the existing test file so I didn't clutter up your project. Would you like me to add the issue's `test.xlsx` file and write a test against that file specifically?

If yes, do you want me to revert the changes to the existing `merged_ranges.xlsx` test file, or keep both tests?